src/content/documentation/develop/developing-extensions-for-firefox-for-android-fennec.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: sidebar
+title: Developing extensions for Firefox for Android (Fennec)
+permalink: /documentation/develop/developing-extensions-for-firefox-for-android-fennec/
+topic: Develop
+tags: [add-ons, beginner, guide, mobile, webextensions]
+contributors:
+  [caitmuenster, rebloor, juraj, mdnwebdocs-bot, ExE-Boss, Ding-Fan, andrewtruongmoz]
+last_updated_by: caitmuenster
+date: 2020-08-26
+---
+
+{% capture page_hero_banner_content %}
+
+# Developing extensions for Firefox for Android < version 68
```

Comment:
"version ≤ 68" or "version 68 and earlier"

instead of "< version 68".

Similarly for everything else in this PR that references Fennec.